kokoro/linux/dockerfile/test/csharp/Dockerfile

```diff
@@ -33,7 +33,7 @@ RUN apt-get update && apt-get install -y libunwind8 libicu63 && apt-get clean
 RUN wget -q https://dot.net/v1/dotnet-install.sh && \
     chmod u+x dotnet-install.sh && \
     ./dotnet-install.sh --version 2.1.802 && \
-    ./dotnet-install.sh --version 5.0.102 && \
+    ./dotnet-install.sh --version 5.0.202 && \
```

Comment:
is this required? I will need to rebuild the base dockerfile if so for the tests to pass.

Reply:
Yes. SDK version needs to be 5.0.200 or later to embed all the required metadata on build.